pulp_ansible/app/serializers.py

```diff
@@ -132,7 +133,18 @@ def validate(self, data):
         data = super().validate(data)
 
         if data.get("requirements_file"):
-            parse_collections_requirements_file(data["requirements_file"])
+            parsed_file = parse_collections_requirements_file(data["requirements_file"])
```

Comment:
I'm confused, I didn't think we would be validating the requirements file since it doesn't contain urls typically. https://docs.ansible.com/ansible/devel/galaxy/user_guide.html#install-multiple-collections-with-a-requirements-file

I thought when a requirements file is being used, the `remote.url` would be validated.

Reply:
I'm validating `source` which is a URL